sdk/search/search-documents/swagger/Service.md

```diff
@@ -218,3 +218,58 @@ directive:
     transform: >
       $["x-ms-client-name"] = "maxPageLength"
 ```
+
+### Make credentials of SearchIndexerDataSource/Credentials to ConnectionString
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.definitions.SearchIndexerDataSource.properties
+    transform: >
+      delete $['credentials']
+  - from: swagger-document
+    where: $.definitions.SearchIndexerDataSource
+    transform: >
+      $['properties']['connectionString'] = {};
```

Comment:
are you not able to do this part inside the previous transform?

Reply:
Done